src/hotspot/share/opto/superword.cpp

```diff
@@ -1756,6 +1839,32 @@ void SuperWord::combine_packs() {
     }
   }
 
+  if (_do_vector_loop) {
+    // Since we did not enforce exact alignment of the packsets, we only know that there
+    // is no dependence with distance 1, because we have checked independent(s1, s2) for
+    // all adjacent memops. But there could be a dependence of a different distance.
+    // Hence: remove the pack if there is a dependence.
+    for (int i = 0; i < _packset.length(); i++) {
+      Node_List* p = _packset.at(i);
+      if (p != NULL) {
```

Comment:
There are some more occurences in your changes.

```suggestion
      if (p != nullptr) {
```